README.md

```diff
@@ -25,10 +25,19 @@ You can subscribe to receive release updates by following these steps:
 
 ## Requirements
 
-* The minimum supported Android version is 5.0 (SDK INT 21)
-* Your app must be compiled with at least an API version **29**
+* `minSdkVersion`: **21** (Android 5.0)
+* `compileSdkVersion`:**29**  
+* Java 8 language feature support
+* [Android X](https://developer.android.com/jetpack/androidx/) as of version SDK 2.0.0
 * A Beacon Id created on [Help Scout](https://secure.helpscout.net/settings/beacons/)
 
+### Supported platform and language versions
+
+* Android 8.0 to 11.0
```

Comment:
I know that internally we're only going to be offering support for Android 8.0 and up and even though the SDK can still be installed in Android 5.0 devices, we're not actively verifying that there aren't any breaking changes in there. However, if I was reading this README and saw "`minSdkVersion`: **21** (Android 5.0)" and then this I'd be a little bit confused.

Maybe we should put a clarifying note in here.

Reply:
Hey @aliripe I've added your suggestion. Thanks!